app/controllers/idv/review_controller.rb

```diff
@@ -90,6 +90,10 @@ def init_profile
       idv_session.cache_encrypted_pii(password)
       idv_session.complete_session
 
+      if idv_session.address_verification_mechanism == 'gpo'
+        analytics.idv_gpo_address_letter_requested(enqueued_at: Time.zone.now)
```

Comment:
Just had the thought that we may want to add a `resend` attribute or something like that to these. That way we can filter by whether a letter is being resent. That could potentially be a different story though.